packages/usdk/lib/create.mjs

```diff
@@ -342,7 +342,7 @@ const uploadImage = async (file, {
 
 export const create = async (args, opts) => {
   // args
-  let dstDir = args._[0] ?? '';
+  let dstDir = args._[0] ?? cwd;
```

Comment:
Should we keep it empty and then check emptiness in the name replacer below? I don't necessarily think we should default to the cwd for creation; if the user really wanted this they could use `.`